src/py42/clients/detectionlists/departing_employee.py

```diff
@@ -7,110 +7,88 @@
 class DepartingEmployeeClient(BaseClient):
     """A client for interacting with Code42 Departing Employee APIs."""
 
-    _uri_prefix = u"/svc/api/v1/departingemployee/{0}"
+    _uri_prefix = u"/svc/api/v2/departingemployee/{0}"
 
     def __init__(self, session, user_context):
         super(DepartingEmployeeClient, self).__init__(session)
         self._user_context = user_context
 
-    def create(
-        self, username, tenant_id=None, notes=None, departure_epoch=None, cloud_usernames=None
-    ):
+    def add(self, user_id, departure_date):
         """Adds a user to Departing Employees.
-        `REST Documentation <https://ecm-default.prod.ffs.us2.code42.com/svc/swagger/index.html#/DepartingEmployeeCase/DepartingEmployeeCase_Create>`__
+        `REST Documentation <https://ecm-east.us.code42.com/svc/swagger/index.html?urls.primaryName=v2#/>`__
 
         Args:
-            username (str): The username of the departing employee.
-            tenant_id (str, optional): The identifier of the Customer tenant the user is in.
-                Defaults to None (the tenant_id of SDK authorization user will be used).
-            notes (str, optional): Optional descriptive information. Defaults to None.
-            departure_epoch (int, optional): Employee departure date as POSIX timestamp. Defaults
-                to None.
-            cloud_usernames (list, optional): List of alternate usernames for this user from
-                external data sources being searched by Forensic File Search (Google Drive, Box,
-                OneDrive, Gmail, Office 365). Defaults to None.
+            user_id (str or int): Id of the departing employee.
```

Comment:
I would clarify that this is the uid of the Code42 user.

Reply:
Changed.